coalib/parsing/InvalidFilterException.py

```diff
@@ -1,2 +1,13 @@
 class InvalidFilterException(Exception):
-    pass
+    def __init__(self, filter_name, available_filters):
+        self.filter_name = filter_name
+        self.available_filters = available_filters
+        if self.available_filters is None:
+            message = 'Invalid Filter'
```

Comment:
IMO this condition is necessary. Its very unlikely.
If you still want to keep it, update the message stating no filters available.